src/commcare_cloud/commands/ansible/service.py

```diff
@@ -59,12 +59,12 @@ def __init__(self, environment, ansible_context):
         self.environment = environment
         self.ansible_context = ansible_context
 
-    def run(self, action, host_pattern=None, process_pattern=None):
+    def run(self, action, env_name, host_pattern=None, process_pattern=None):
         if action == 'help':
             self.print_help()
             return 0
         elif action == 'logs':
-            print("Logs can be found at:\n{}".format(self.log_location))
+            print("Logs can be found at:\n{}".format(self.log_location.format(env_name)))
```

Comment:
nice. You could also keep it as `{env}` below and use `.format(env=env_name)` here. That way someone reading just that line (or producing a similar one later) knows just by looking at it what's intended.

Reply:
Cool, didn't realize I could do that; https://github.com/dimagi/commcare-cloud/pull/2055/commits/90509753db851fc32509115dd32791c1087c2c19